src/main/java/decodex/data/exception/RecipeManagerException.java

```diff
@@ -0,0 +1,14 @@
+package decodex.data.exception;
+
+/**
+ * This exception is raised when an error during recipe manager execution occurs.
+ */
+public class RecipeManagerException extends Exception {
+
+    public static final String EMPTY_RECIPELIST_MESSAGE = "[x] There is nothing to be removed.";
```

Comment:
Perhaps the message can be "The recipe is currently empty. There is no module to be removed." to be more descriptive.